python/instrumentation/openinference-instrumentation-crewai/README.md

```diff
@@ -45,7 +44,6 @@ trace_provider = TracerProvider()
 trace_provider.add_span_processor(SimpleSpanProcessor(OTLPSpanExporter(endpoint)))
 
 CrewAIInstrumentor().instrument(tracer_provider=trace_provider)
-LangChainInstrumentor().instrument(tracer_provider=trace_provider)
```

Comment:
Good looking out here. I think CrewAI uses LiteLLM under the hood? I guess we should swap out LiteLLMInstrumentor here from what I understand

Reply:
hm, good call. CrewAI Instrumentor has custom wrappers that log in a really similar way to how a Crew does. Not sure if the logs will look as clean as they do today though. Will check